prow/gcpproxy-coverage.sh

```diff
@@ -46,12 +46,11 @@ echo '======================================================='
 PUBLIC_DIRECTORY=""
 case "${JOB_TYPE}" in
   "presubmit")
-    # Store in directory with job id.
-    # IMPORTANT: Default to a some directory to prevent messy copy on unset var.
-    PUBLIC_DIRECTORY="${PROW_JOB_ID:-tmp}"
+    # Store in directory with the SHA for each presubmit run.
+    PUBLIC_DIRECTORY=$(get_tag_name)
     ;;
   "periodic")
-    # Overwrite global directory with latest coverage.
+    # Overwrite global directory with latest coverage for all continuous runs.
```

Comment:
it seems that we don't need code-coverage run for periodic tests since presubmit tests should have run that for each sha.

Should we disable it in period run if it is expensive( taking too long, using too much memory?  If it is cheap, just keep it.

Reply:
I want it to run on periodic tests so that we can display our C++ unit test coverage at HEAD. You should be able to just visit a single URL and see the latest coverage at any time, similar [to this](https://storage.googleapis.com/esp-v2-coverage/local-run/third_party/tools/coverage/coverage_tests/index.html).

If we just keep presubmits, then it'll be hard to sync that to the `latest` URL since jobs can run out-of-order, 2 jobs can run at once, etc.

Shouldn't be expensive, especially with the remote cache. This might be one of the fastest jobs we have actually.